project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/responsive.js

```diff
@@ -14,7 +14,7 @@
     var lastIsDesktop = null;
 
     Shopsys.responsive.isDesktopVersion = function () {
-        return $(window).width() >= Shopsys.responsive.LG;
+        return window.innerWidth >= Shopsys.responsive.VL;
```

Comment:
Im not sure about these changes. The `innerWidth` gets width of element with paddings, not border or margin. In description you were talking about scrollbar, so i dont know how `innerWidth` would help in this one? Also why is desktop's pixel definition `980` and not `769` as before?